test/unique.ts

```diff
@@ -79,13 +79,15 @@ test('Update user 2 with unique email', async() => {
 })
 
 
-test('Update non-unique property', async() => {
```

Comment:
Do we still want the old non-unique property test case. i.e. should we add a new test case rather than replace the old one?